searxng_extra/update/update_languages.py

```diff
@@ -224,7 +233,7 @@ def write_languages_file(languages):
 
 if __name__ == "__main__":
     load_engines(settings['engines'])
-    engines_languages = fetch_supported_languages()
-    all_languages = join_language_lists(engines_languages)
-    filtered_languages = filter_language_list(all_languages)
-    write_languages_file(filtered_languages)
```

Comment:
The master branch version is more readable IMO.

Reply:
Yes, but there is a name collision:

```
make test.pylint
TEST      [pylint] $PYLINT_FILES
************* Module searxng_extra.update.update_languages
searxng_extra/update/update_languages.py:35:4: W0621: Redefining name 'engines_languages' from outer scope (line 236) (redefined-outer-name)
searxng_extra/update/update_languages.py:65:24: W0621: Redefining name 'engines_languages' from outer scope (line 236) (redefined-outer-name)
searxng_extra/update/update_languages.py:127:25: W0621: Redefining name 'all_languages' from outer scope (line 237) (redefined-outer-name)
searxng_extra/update/update_languages.py:140:4: W0621: Redefining name 'filtered_languages' from outer scope (line 238) (redefined-outer-name)
```

To avoid name collision, I added a leading underline to the names.